packages/amplify-category-api/Readme.md

```diff
@@ -7,6 +7,7 @@ The following table lists the current set of commands supported by the Amplify A
 | Command              | Description |
 | --- | --- |
 | amplify api add | Takes you through steps in the CLI to add an API resource to your backend.   |
+| amplify api add-datasource | Takes you through the steps in the CLI to import a data source to an existing GraphQL API resource. |
```

Comment:
to import an already existing Aurora Serverless data source?

Reply:
done - switched to appropriate wording.